dance/transforms/graph/feature_feature_graph.py

```diff
@@ -15,11 +16,12 @@ def __init__(self, threshold: float = 0.3, *, normalize_edges: bool = True, **kw
         self.threshold = threshold
         self.normalize_edges = normalize_edges
 
-    def __call__(self, data):
+    def __call__(self, data, dist_func_id=0):
```

Comment:
@xingzhongyu Is this work in progress? Also, set options at `__init__` instead of `__call__`; use string instead of id (do a internal conversion from string to id if nessary instead of having user to specify ids).

Reply:
I have changed it